server/schedulers/metrics.go

```diff
@@ -125,4 +142,6 @@ func init() {
 	prometheus.MustRegister(opInfluenceStatus)
 	prometheus.MustRegister(tolerantResourceStatus)
 	prometheus.MustRegister(hotPendingStatus)
+	prometheus.MustRegister(hotPendingSum)
```

Comment:
Do these metrics need to be reset once the leader is transferred?

Reply:
After remove store, we also need to clear the corresponding store information.